docs/Current_Code.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import numpy
+from pyfirmata import Arduino, util
+import time
+from time import sleep
+from pyfirmata import INPUT, OUTPUT, PWM
+import serial
+import re
+import matplotlib.pyplot as plt
+
+class myArduinoBoard(object):
+    def __init__(self,portname = '/dev/tty.usbmodem14201'):
+        self.port = portname
+
+    def setup_arduino(self):
+        self.board = Arduino(self.port)
+        self.arduino = serial.Serial(self.port, 9600)
+        sleep(0.02)  # The synchronization between the Arduino board and pyFirmata requires some time. 1 = 1 second
+        self.analog_d9p = self.board.get_pin('d:9:p')  # Same as board.digital[9].mode = PWM
+        self.board.analog[0].mode = INPUT  # voltage
```

Comment:
This is an ugly construct and I am not sure I understand what is happening.  ``INPUT`` is imported from ``pyfirmata``, but what is it?  A global variable?  or a hook to an input stream?  But if we give ``INPUT`` to both ``analog[0]`` and ``analog[2]`` it seems as if we are making them the same somehow.  I am sure there is some more pythonic way of doing what you are trying to do.

Reply:
This is to set the mode of each of these pins to "input" (as opposed to setting them to output). This is done in order for the program to recognize if it should be receiving or sending data to this pin.